zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/authorization/AuthorizationCheckBehaviorTest.java

```diff
@@ -801,6 +801,7 @@ public void shouldBeAuthorizedThroughMultipleMappings() {
             .mapping()
             .newMapping(firstClaimName)
             .withClaimValue(firstClaimValue)
+            .withId(UUID.randomUUID().toString())
```

Comment:
❓ Do you have an issue to actually use this mapping id when retrieving authorizations? Currently it seems we're still doing it using the mapping key and I fear we are running out of sync with the parallel streams of work that are happening.

Reply:
For example I'd expect that when creating authorization on line 826 we'd provide the id now, instead of the key as a string. This comment applies to other methods as well.